kie-server-parent/kie-server-wars/kie-server/pom.xml

```diff
@@ -124,6 +142,11 @@
     </dependency>
     <dependency>
       <groupId>org.kie.server</groupId>
+      <artifactId>kie-server-services-swagger</artifactId>
+      <!-- TODO Exclusions? -->
```

Comment:
Probably not :-) I think I put that there because I was using the swagger-jaxrs-jersey dependency at first, which brought in all sorts of jersey transitive dependencies .... I think we now simply depend on swagger-jars.